Core/Tests/Speckle.Core.Tests.Integration/Credentials/UserServerInfoTests.cs

```diff
@@ -15,17 +15,37 @@ public async Task Setup()
   }
 
   [Test]
-  public async Task GetServerInfo()
+  public async Task IsFrontEnd2True()
   {
-    Uri serverUrl = new(acc.serverInfo.url);
-    ServerInfo result = await AccountManager.GetServerInfo(serverUrl);
+    ServerInfo result = await AccountManager.GetServerInfo("https://app.speckle.systems/");
+
+    Assert.That(result, Is.Not.Null);
+    Assert.That(result.frontend2, Is.True);
+  }
 
-    Assert.That(result.url, Is.EqualTo(acc.serverInfo.url));
-    Assert.That(result.name, Is.Not.Null);
-    Assert.That(result.company, Is.Not.Null);
+  [Test]
+  public async Task IsFrontEnd2False()
+  {
+    ServerInfo result = await AccountManager.GetServerInfo("https://speckle.xyz/");
+
+    Assert.That(result, Is.Not.Null);
     Assert.That(result.frontend2, Is.False);
   }
 
+  /// <remarks>
+  /// We get ServerInfo from "http://localhost:3000/graphql",
+  /// Then we mutate the `frontend2` property of ServerInfo by trying to fetch header from "http://localhost:3000/",
+  /// This is not doable in local server because there is no end-point on this to ping.
+  /// This is a bad sign for mutation.
+  /// </remarks>
```

Comment:
Doesn't the local server do the opposite? i.e. it is the frontend that proxies the backend. So maybe it should be `localhost:8080/graphql` for it to work? 🤔

But just the fact that we're discussing this indicates that relying on this header may not be as "reliable" as we were lead to believe...

Reply:
On local server we don't have proxy for frontend. i.e. we run FE2 on `localhost:8081`. So we can't get this header from same server url which is `localhost:3000` on our integration tests
![image](https://github.com/specklesystems/speckle-sharp/assets/45078678/d58937f6-9a9a-4847-96f6-3b9cde5b5d81)